<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index_passwordCh</class>
 <widget class="QDialog" name="Index_passwordCh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res_Index.qrc">
    <normaloff>:/icons/icons/food/21.png</normaloff>:/icons/icons/food/21.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,5" columnminimumwidth="10,10">
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请输入原密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="OriPaEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;账号名&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="NewPaEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请再次输入新密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="IDEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="NewPaAgainEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请输入新密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="OKButton">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res_Index.qrc"/>
 </resources>
 <connections/>
</ui>
